kevlar/call.py

```diff
@@ -73,15 +73,29 @@ def dedup(callstream):
         if call.seqid not in calls:
             calls[call.seqid] = defaultdict(set)
         calls[call.seqid][call.position].add(call)
-    for seqid in calls:
-        for position in calls[seqid]:
+    for seqid in sorted(calls):
+        for position in sorted(calls[seqid]):
```

Comment:
The `merge_adjacent` function requires that the variants are sorted by genomic position.